### ISSUE TITLE
Colors for Remote Play

### ISSUE THREAD

**Issue by toddmedema** [ExpeditionRPG/expedition-app#472]
@smartin015 Researched generating a colorblind-friendly color scheme, came up with a few options:

- http://colorbrewer2.org/#type=diverging&scheme=RdYlBu&n=6
![capture](https://user-images.githubusercontent.com/775657/30487196-c002e010-9a00-11e7-9ba5-53857cdcde1f.PNG)
#d73027, #fc8d59, #fee090, #e0f3f8, #91bfdb, #4575b4
- http://www.somersault1824.com/tips-for-designing-scientific-figures-for-color-blind-readers/
![foo](https://user-images.githubusercontent.com/775657/30487711-40353e44-9a02-11e7-8396-9d1cd2e66cc2.png)
- Using shapes in addition to colors. For example, we could create player 1-6 SVGs, and then also color-code them. This would let us use slightly more thematic colors (though I think option 2 seems fine), but it would also generally improve rapid visual distinction (especially if we're thinking of doing a bottom bar to show player connection status... we wouldn't be able to grey them out on disconnect if their only unique identification is their color).
  - Initial thought was to just re-use existing Expedition icons, but that could get confusing (ie if your mage gets randomly assigned the melee icon). We could either let people pick their icon (cute, but more code, and more things for people to do to set up), or use a different set of icons (horoscope symbols? ancient greek letters?) without in-game meaning.
  - TL;DR maybe we start with colorblind friendly colors, and add icons later / if other player recognition turns out to be an issue

**Comment by smartin015** [comment]
Yeah, I vote we follow your TL;DR. Would still be neat to colorize the adventurer icon, but we could even start with just a square or circle as a proof of concept.

**Comment by toddmedema** [comment]
Probs circles as that's already what we use for finger indicators, and that tends to be the standard way of showing status (a la Google Chat / AIM / Facebook Messenger

**Comment by toddmedema** [comment]
Moved colors to main RP issue: https://github.com/ExpeditionRPG/expedition-app/issues/446
